pkg/subscriptions/subscriptions.go

```diff
@@ -39,9 +38,18 @@ type subscriptionData struct {
 func (s subscriptionData) saveTo(dir string) error {
 	path := filepath.Join(dir, s.name)
 	if err := os.MkdirAll(filepath.Dir(path), s.dirMode); err != nil {
```

Comment:
Do we need to worry about permissions of parent directories that are created by this?

At least in principle, it seems that the mounts file can contain ~arbitrary destination paths.

Applies to all the `MkdirAll` calls.

Reply:
Yes, changed the code to reflect that.